feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,1025 @@
+	v4 := flowV4.Packet().Add().Ipv4()
+	v4.Src().Increment().SetStart(srcV4.ip).SetCount(v4Count)
+	v4.Dst().Increment().SetStart(dstV4.ip).SetCount(ipRange)
+	return flowV4
+}
+
+// installGRIBIRoutes configure route using gRIBI client
+func installGRIBIRoutes(t *testing.T, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice, top gosnappi.Config) {
+	t.Helper()
+	ctx := context.Background()
+	gribic := dut.RawAPIs().GRIBI(t)
+	client := fluent.NewClient()
+	client.Connection().WithStub(gribic).WithPersistence().WithInitialElectionID(12, 0).
+		WithRedundancyMode(fluent.ElectedPrimaryClient).WithFIBACK()
+	client.Start(ctx, t)
+	defer client.Stop(t)
+	gribi.FlushAll(client)
+	client.StartSending(ctx, t)
+	gribi.BecomeLeader(t, client)
+
+	tcArgs := &testArgs{
+		ctx:    ctx,
+		client: client,
+		dut:    dut,
+		ate:    ate,
+		top:    top,
+	}
+	const (
+		nhgID1 uint64 = 100
+		nhID1  uint64 = 1001
+	)
+
+	t.Logf("An IPv4Entry for %s is pointing to ATE LAG2 via gRIBI", pfx4AdvV4.ip+"/24")
+	nh := fluent.NextHopEntry().
+		WithNetworkInstance(deviations.DefaultNetworkInstance(tcArgs.dut)).
+		WithIndex(nhID1).
+		WithIPAddress(agg2.ateIPv4)
+
+	nhg := fluent.NextHopGroupEntry().
+		WithNetworkInstance(deviations.DefaultNetworkInstance(dut)).
+		WithID(uint64(nhgID1)).
+		AddNextHop(nhID1, 1)
+
+	tcArgs.client.Modify().AddEntry(t, nh, nhg)
+
+	tcArgs.client.Modify().AddEntry(t,
+		fluent.IPv4Entry().
+			WithPrefix(pfx4AdvV4.ip+"/24").
+			WithNetworkInstance(deviations.DefaultNetworkInstance(dut)).
+			WithNextHopGroup(nhgID1))
+}
+
+// startTraffic start traffic on ATE
+func startTraffic(t *testing.T, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice, top gosnappi.Config) {
+	t.Helper()
+	capturePktsBeforeTraffic(t, dut, dutPortList)
+	time.Sleep(10 * time.Second)
+	ate.OTG().StartTraffic(t)
+	time.Sleep(time.Minute)
+	ate.OTG().StopTraffic(t)
+	time.Sleep(time.Second * 40)
+	otgutils.LogFlowMetrics(t, ate.OTG(), top)
+	otgutils.LogLAGMetrics(t, ate.OTG(), top)
+	otgutils.LogPortMetrics(t, ate.OTG(), top)
+}
+
+// capturePktsBeforeTraffic capture the pkts before traffic on DUT Ports
+func capturePktsBeforeTraffic(t *testing.T, dut *ondatra.DUTDevice, dutPortList []*ondatra.Port) {
+	rxPktsBeforeTraffic = map[*ondatra.Port]uint64{}
+	txPktsBeforeTraffic = map[*ondatra.Port]uint64{}
+	for _, port := range dutPortList {
+		rxPktsBeforeTraffic[port] = gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).Counters().InPkts().State())
+		txPktsBeforeTraffic[port] = gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).Counters().OutPkts().State())
+	}
+}
+
+// verifyTrafficFlow verify the each flow on ATE
+func verifyTrafficFlow(t *testing.T, ate *ondatra.ATEDevice, flows []gosnappi.Flow) bool {
+	if flows[0].Name() == "pfx1ToPfx4" {
+		rxPkts := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(flows[0].Name()).Counters().InPkts().State())
+		if got := rxPkts / 100; got == 0 {
+			t.Logf("Packet Dropped, LossPct for flow %s: got %d, want 0 packet", flows[0].Name(), got)
```

Comment:
I dont see need to calculate % when we are checking only RX packets

Reply:
This is just to verify no single Packet is receiving for PFX1TOPFX4

Reply:
To be more clear then, you could change this to be:

`if got := rxPkts ; got == 0 `

Reply:
Done